From 9ce534491aea9b7975cfee5064423d0b850d8002 Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@openwide.fr>
Date: Sun, 9 Aug 2015 11:20:00 +0200
Subject: [PATCH] vipw: Remove pre-ANSI compiler support

__P() is used for compatibility with old K&R C compilers. With
ANSI C this macro has no effect.

This fixes a compilation error with musl libc because of undeclared
__P.

Ref:
https://lists.samba.org/archive/samba-technical/2015-June/108042.html

Signed-off-by: Romain Naour <romain.naour@openwide.fr>
---
 login-utils/vipw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/login-utils/vipw.c b/login-utils/vipw.c
index 668f4d8..e4c803b 100644
--- a/login-utils/vipw.c
+++ b/login-utils/vipw.c
@@ -85,7 +85,7 @@ int program;
 char orig_file[FILENAMELEN];	/* original file /etc/passwd or /etc/group */
 char *tmp_file;			/* tmp file */
 
-void pw_error __P((char *, int, int));
+void pw_error (char *, int, int);
 
 static void copyfile(int from, int to)
 {
-- 
2.4.3

